packages/web-components/fast-foundation/.storybook/main.cjs

```diff
@@ -24,6 +24,13 @@ module.exports = {
             hints: false,
         };
         config.module.rules = [
+            {
+                test: /\.svg$/,
+                loader: "svg-inline-loader",
+                options: {
+                    removeSVGTagAttrs: false,
+                },
+            },
```

Comment:
@radium-v do you know if we're setup on a version where we can leverage the statics/builtins from storybook?

Reply:
I considered updating to use the assets model but this was a known entity.